lib/service_provider.rb

```diff
@@ -49,20 +46,12 @@ def cert
     @cert ||= File.read("#{cert_dir}#{host_attributes['cert']}.crt")
   end
 
-  def block_encryption
-    host_attributes['block_encryption'] || DEFAULT_ENCRYPTION
-  end
-
-  def key_transport
-    'rsa-oaep-mgf1p'
-  end
-
   def encryption_opts
-    return nil unless encrypt_responses?
+    return nil if FeatureManagement.saml_encryption_disabled?
```

Comment:
or are you trying to skip having a config line at all, unless wanting to disable the feature?